frontend/tet/admin/azure-pipelines-devtest.yml

```diff
@@ -7,7 +7,7 @@ trigger:
   paths:
     include:
     - frontend/tet/admin/**
-    - frontend/*
+#    - frontend/*

```

Comment:
I think it should trigger all services when
- `frontend/*` (something under frontend folder changes (ike package.json)
- `frontend/shared/*` ( any common code changes)

Reply:
Alright was concerned about Node CPU spikes when all pipelines start running at the same time, but if its needed lets leave it like it is then.